doc/en/developer/source/policies/service_providers.rst

```diff
@@ -0,0 +1,91 @@
+.. _service_providers:
+
+Service Providers
+=================
+
+Service providers are an important part of GeoServer community, thank you for helping GeoServer reach a larger audience. Just as open source is a part of your success, your participation and investment is fundamental to the GeoServer project's sustainability.
+
+The PSC manages:
+
+* `GeoServer Commercial Support <http://geoserver.org/support/>`__ (geoserver.org)
+* `Service Providers List <https://www.osgeo.org/service-providers/?p=geoserver>`__ (osgeo.org)
+  
+  Available from `project page <https://www.osgeo.org/projects/geoserver/>`__ from *view all service providers* link.
+
+In addition:
+
+* Project `sponsorship <https://github.com/geoserver/geoserver/wiki/Sponsor>`__ is another way to increase visibility, with specific activities available to sponsor throughout the year. Financial support is also recognized on the `osgeo sponsor <https://www.osgeo.org/sponsors/>`__ page.
+
+Service Provider checklist
+--------------------------
+
+To recognizing experienced providers and core contributors consistently we have developed the following checklist.
+
+Core contributors:
```

Comment:
Core contributors are doing all of these

Reply:
These three all have an aspect of taking responsibility for the geoserver project so I am marking them core.